app/scripts/services/UsersApiRequests.js

```diff
@@ -0,0 +1,67 @@
+(function() {
+  function UsersApiRequests($http, $cookies, ListsApiRequests) {
+
+    var UsersApiRequests = {};
+
+    UsersApiRequests.user_signed_in = function() {
```

Comment:
`userSignedIn` or `isUserSignedIn` would be even better.

Reply:
Actually I"m confused by this one, it's called `user_signed_in` but it appears to set the users list. The naming and the functionality don't line up.